selectors_ton.yml

```diff
@@ -0,0 +1,8 @@
+selectors:
+  -239:
+    name: ton-mainnet
+    selector: 16448340667252469081
+  -3:
+    name: ton-testnet
+    selector: 1399300952838017768
+
```

Comment:
For local Devnet we can use `-217` as it is the [mylocalton-docker default](https://github.com/neodix42/mylocalton-docker/blob/main/docker/scripts/start-genesis.sh#L53).

Reply:
Is this what we going to run locally for e2e test ?

Reply:
Yes, this is what we will try to use for local e2e tests.

It's a recommendation we got from the TON team.